### PR TITLE
completed actors db

### PR DESCRIPTION
**Pull request by ksullivan74**
in database.js added actors data and created export function.

### CODE REVIEW

**Review by VogelMatt — CHANGES_REQUESTED** [review]

File: scripts/database.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+  },
+  {
+    id:6,
+    name: "Ingrid Bergman"
+  }
+]
 
 }
 
 export const getShows = () => {
```
[the remark this answers is not in the document]
Comment (ksullivan74) [reply scripts/database.js]: did it

**Review by pino-fu — APPROVED** [review]
nice comment..

**Review by VogelMatt — APPROVED** [review]
🎉

**Review by jhbabbitt — APPROVED** [review]
This is also a nice comment

**Review by NelsonTaylorLumpkin — APPROVED** [review]
Lerkin gerdddd

**Pull request merged**